packages/keystrokes/src/keystrokes.ts

```diff
@@ -419,20 +423,11 @@ export class Keystrokes<
 
   private _tryReleaseSelfReleasingKeys() {
     for (const activeKey of this._activeKeyPresses) {
-      let isSelfReleasingKey = false
       for (const selfReleasingKey of this._selfReleasingKeys) {
         if (activeKey.key === selfReleasingKey) {
-          isSelfReleasingKey = true
-          break
+          this._releaseKey(activeKey.event)
         }
       }
-      if (!isSelfReleasingKey) {
-        return
-      }
-    }
-
-    for (const activeKey of this._activeKeyPresses) {
-      this._handleKeyRelease({ key: activeKey } as any)
```

Comment:
This line had an error where a `{ key: activeKey }` object was being constructed.  This resulted in a crash.

Additionally, the `_tryReleaseSelfReleasingKeys` method seemed to be repeatedly calling `handleKeyRelease` (and thus itself).  Instead, it simply loops over the self releasing keys and releases them.  There is no longer any recursion in this code path.

Reply:
Indeed, this implementation is a bit strange. The double loops was initially coded to bail out of the loop early if not self releasing keys are found, but looking at it again, it simply goes through the whole set of active keys anyway.

This feature was actually was created at a replacement for KeyboardJS's killKeys. The idea being if a key is released it releases all other active keys too. This was implemented in keyboardJS because of browser bugs where some keys would not fire a release event correctly, or would prevent other keys from firing release events. But with my meta key fix in #10 at the binder level, it might not be necessary to have this feature at all, or at least for the purpose I added it for.

You've changed this to be (understandably) what the name implies, keys that release themselves. Do you think we should keep it - are you wanting to use it, and if so what for?

Reply:
Yea I was trying to use it for the apple "command - Z" issue - where "Z" wouldn't get a keyup...So if you've found a different workaround that's great.